rpc/lib/rpc_test.go

```diff
@@ -326,6 +353,22 @@ func TestWSHandlesArrayParams(t *testing.T) {
 	}
 }
 
+// TestWSClientPingPong checks that a client & server exchange pings
+// & pongs so connection stays alive.
+func TestWSClientPingPong(t *testing.T) {
+	if testing.Short() {
+		t.Skip("skipping ping pong in short mode")
+	}
+
+	cl := client.NewWSClient(tcpAddr, websocketEndpoint)
+  cl.SetLogger(log.TestingLogger())
+	_, err := cl.Start()
+	require.Nil(t, err)
+	defer cl.Stop()
+
+	time.Sleep(35 * time.Second)
```

Comment:
the problem here is that the server has a hard coded timeout:

```
	wsReadTimeoutSeconds  = 30 // connection times out if we haven't received *anything* in this long, not even pings.
```

and if we wanna test this we need to either wait >30sec or make it an option